### ISSUE TITLE
Cli interface to help you detect all components without Accessibility attributes ?

### ISSUE THREAD

**Issue by pligor** [aryella-lacerda/react-native-accessibility-engine#255]
The issue is that one already has a huge React Native application and wants to understand which components in which files and folders have Accessibility Identifiers and which are lacking those.

Of course one does not want to put Accessibility Identifiers in all elements therefore it would be great to prioritize accordingly. Namely to have the clickable elements be listed first and then follow some other elements.

This is more of a proposal of a cli tool that would scan the entire folder of a React Native project and would pinpoint which, folders, files and components which are lacking Accessibility elements and would therefore create a kind of a todo list for the dev team to add all those missing Accessibility identifiers.

Makes sense ?
